.devcontainer/setup.py

```diff
@@ -90,8 +90,8 @@ def setup():
     """Runs various setup commands."""
     run_commands([
         # allow connecting from any host, needed to connect from host to PG running in docker
-        "sed -i -e \"s/#listen_addresses = \'localhost\'/listen_addresses = \'*\'/\" /srv/postgres14/postgresql.conf",
-        "echo 'host    all         all     0.0.0.0/0     trust' >> /srv/postgres14/pg_hba.conf",
+        "sed -i -e \"s/#listen_addresses = \'localhost\'/listen_addresses = \'*\'/\" /srv/postgres/postgresql.conf",
+        "echo 'host    all         all     0.0.0.0/0     trust' >> /srv/postgres/pg_hba.conf",
```

Comment:
Not sure why we are reverting it here. Postgres in devcontainer was upgraded recently (https://github.com/percona/pmm/pull/1051). 

Also, running `SHOW config_file` here gives this:

```
[root@pmm-managed-server pmm]# psql -U postgres -c 'SHOW config_file'
           config_file
---------------------------------
 /srv/postgres14/postgresql.conf
(1 row)

[root@pmm-managed-server pmm]#
```

Reply:
I see that Nikita raised similar concern above already.